native/src/App.tsx

```diff
@@ -36,17 +40,36 @@ const linking: LinkingOptions = {
       [REDIRECT_ROUTE]: '*'
     }
   },
-  getStateFromPath: path => {
-    return {
-      index: 0,
-      routes: [
-        {
-          name: REDIRECT_ROUTE,
-          params: {
-            url: `https://${path}`
-          }
+  getStateFromPath: path => ({
+    index: 0,
+    routes: [
+      {
+        name: REDIRECT_ROUTE,
+        params: {
+          url: `https://${path}`
         }
-      ]
+      }
+    ]
+  }),
+  subscribe: (listener: (url: string) => void) => {
+    const onReceiveURL = ({ url }: { url: string }) => listener(url)
+
+    Linking.addEventListener('url', onReceiveURL)
+
+    const unsubscribeNotification = messaging().onNotificationOpenedApp(() => {
```

Comment:
We could also call this subscript funciton outside of the react-native navigation `subscribe`. 

That way we do not have to use placeholders. We can simply call `navigateToDeepLink` from the callback of onNotificationOpenedApp.

Reply:
Tried many stuff and invested way too much time for this, will keep it as it is. For `navigateToDeepLink` we need the `navigation` prop, so it is not possible in sagas. We also need city code and language code so, not really possible either.